staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope_test.go

```diff
@@ -230,14 +234,15 @@ func TestBackwardsCompatibility(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to initialize envelope transformer: %v", err)
 	}
-	context := value.DefaultContext([]byte(testContextText))
+	ctx := context.Background()
+	dataCtx := value.DefaultContext([]byte(testContextText))
 	originalText := []byte(testText)
 
-	transformedData, err := oldTransformToStorage(envelopeTransformerInst.(*envelopeTransformer), originalText, context)
+	transformedData, err := oldTransformToStorage(envelopeTransformerInst.(*envelopeTransformer), ctx, originalText, dataCtx)
```

Comment:
nit: make ctx the first param

Reply:
ACK, was trying to keep the (receiver, args) pattern